src/relay/transforms/annotate_texture_storage.cc

```diff
@@ -55,15 +57,17 @@ class StorageInfo : private transform::DeviceAwareExprVisitor {
  public:
   StorageInfo() : transform::DeviceAwareExprVisitor(Optional<IRModule>()) {}
 
-  static Map<Expr, Array<String>> GetStorageMap(const Expr& expr) {
+  static Map<Expr, Map<Expr, Array<String>>> GetStorageMap(const Expr& expr) {
```

Comment:
Introducing of more textures scopes will fails some existing tests. I have not seen any changes in the existing adreno tests in the PR. There should be many changes of "global" to "global.texture" scope in test_conv2d_nchw_texture.py